pontoon/tags/models.py

```diff
@@ -0,0 +1,15 @@
+
+from django.db import models
+
+from pontoon.base.models import Project, Resource
+
+
+class Tag(models.Model):
+    slug = models.CharField(max_length=20)
+    name = models.CharField(max_length=30)
+    project = models.ForeignKey(Project, blank=True, null=True)
```

Comment:
What's the purpose of the project field?
To be used in `unique_together`?
Optimization?

Reply:
to create associated tags in the project admin ui

Reply:
well because @flodolo wanted tags to be per-project so they kinda need to be associated